Library/SubscriptionAddOn.cs

```diff
@@ -100,6 +113,20 @@ internal override void ReadXml(XmlTextReader reader)
                         TierType = reader.ReadElementContentAsString();
                         break;
 
+                    case "tiers":
+                        Tiers = new List<Tier>();
+                        while (reader.Read())
+                        {
+                            if (reader.Name == "tiers" && reader.NodeType == XmlNodeType.EndElement)
+                                break;
+
+                            if (reader.NodeType == XmlNodeType.Element && reader.Name == "tier")
```

Comment:
Nitpicky:  It makes for cleaner code to use `else if` if possible to clarify that the branches are exclusive.

Reply:
Good call!